syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "Enums.proto";
import public "NymIDSource.proto";
import public "CredentialSet.proto";

message CredentialIndex {
    optional uint32 version = 1;
    optional string nymID = 2;
    optional CredentialIndexMode mode = 3;
    optional uint64 revision = 4;
    optional NymIDSource source = 5;
    repeated CredentialSet activeCredentials = 6;
    repeated CredentialSet revokedCredentials = 7;
}
